VoxelSniperFabric/src/main/java/com/github/kevdadev/voxelsniperfabric/VoxelSniperFabric.kt

```diff
@@ -53,7 +53,7 @@ class VoxelSniperFabric : ModInitializer, IVoxelsniper {
 
 
         fileHandler = FabricFileHandler()
-        SchematicReader.initialize()
+        DataFolderSchematicReader.Companion.initialize()
```

Comment:
Why do we need .Companion here??

Reply:
Because it won't see it otherwise. I think it's because the fabric impl only knows the compiled java version, which puts statics in companion